pode.build.ps1

```diff
@@ -19,45 +103,156 @@ param(
     $ReleaseNoteVersion,
 
     [string]
-    $UICulture = 'en-US'
-)
+    $UICulture = 'en-US',
 
-# Fix for PS7.5 Preview - https://github.com/PowerShell/PowerShell/issues/23868
-$ProgressPreference = 'SilentlyContinue'
+    [string[]]
+    [ValidateSet('netstandard2.0', 'netstandard2.1', 'netcoreapp3.0', 'netcoreapp3.1', 'net5.0', 'net6.0', 'net7.0', 'net8.0', 'net9.0', 'net10.0')]
+    $TargetFrameworks = @('netstandard2.0', 'net8.0','net9.0'),
+
+    [string]
+    [ValidateSet('netstandard2.0', 'netstandard2.1', 'netcoreapp3.0', 'netcoreapp3.1', 'net5.0', 'net6.0', 'net7.0', 'net8.0', 'net9.0', 'net10.0')]
+    $SdkVersion = 'net9.0'
```

Comment:
I would keep this list simple, and just make it the versions which are supported by .NET, and the version we know Pode works with. Ie, `netstandard2.0`, `net8.0`, and `net9.0` - you can keep `net10.0` if you want, but the others should be dropped, to prevent people trying to build Pode in `netcoreapp3.0` and raising issues.

This applies to these parameters, the switch statements below in `Get-TargetFramework` and `Get-TargetFrameworkName`